### ISSUE TITLE
mysite.xyz/nextcloud gives a 404 error after following the Nextcloud guide

### ISSUE THREAD

**Issue by ericstephen0118** [LukeSmithxyz/landchad#151]
Title. I'm on Debian 10, set up my site, SSH keys, etc. all to the letter of what's written on the site, and followed the Nextcloud guide to the letter with no errors, and still get an nginx 404 page even after triple checking everything and even reinstalling the server to try again. Looks like this was mentioned a few times in closed issues, the latest one being [here](https://github.com/LukeSmithxyz/landchad/issues/107), but they were all closed by the user without posting how they resolved it.

**Comment by tfasano1** [comment]
Shouldn't your instance be pointed at a subdomain? (nextcloud.example.org)

**Comment by ericstephen0118** [comment]
> Shouldn't your instance be pointed at a subdomain? (nextcloud.example.org)

It can be done either way, but the guide only shows you how to set it up for a subdirectory, not a subdomain.

> Go to https://example.org/nextcloud. This will launch the configuration wizard.

**Comment by tfasano1** [comment]
Well try setting it up as its own domain. That is my best guess. Just follow https://landchad.net/nginx and add a new config file with your desired subdomain. Rerun certbot to get https.

**Comment by aaronbjohnson** [comment]
> > Shouldn't your instance be pointed at a subdomain? (nextcloud.example.org)
> 
> It can be done either way, but the guide only shows you how to set it up for a subdirectory, not a subdomain.
> 
> > Go to https://example.org/nextcloud. This will launch the configuration wizard.

Did you ever figure this out? I'm running into the same issue.

**Comment by DokStook** [comment]
I am having the same issue. Any updates from anyone?

**Comment by aaronbjohnson** [comment]
> I am having the same issue. Any updates from anyone?

No updates on my end yet. I'll probably take another stab at it this weekend if I haven't heard anything.

**Comment by tfasano1** [comment]
What is your `root` directive in your nginx config file for nextcloud? Is it the correct folder?

**Comment by tfasano1** [comment]
> Yes, mine is `/var/www/nextcloud`

Ok and is your nextcloud pointed at its own subdomain or a subdirectory ?

**Comment by tfasano1** [comment]
Did you symbolic link `nextcloud` into `/etc/nginx/sites-enabled` ?
I've never installed nextcloud this way. It seems to me, the next step is to contact the guy who wrote the guide -> https://matthewevan.xyz/contact.html

**Comment by aaronbjohnson** [comment]
> Did you symbolic link `nextcloud` into `/etc/nginx/sites-enabled` ? I've never installed nextcloud this way. It seems to me, the next step is to contact the guy who wrote the guide -> https://matthewevan.xyz/contact.html

Yes I made the symbolic link to sites-enabled. Thanks for your help. I'll reach out to the writer.

**Comment by LunaGNUisance** [comment]
I installed Nextcloud on a subdomain and just used this config, uncommenting php-fpm at the top and using my own URL: https://docs.nextcloud.com/server/13/admin_manual/installation/nginx.html#nextcloud-in-the-webroot-of-nginx

I no longer get a 403 and it redirects me to login.

**Comment by Rudy-MS** [comment]
Would like to say that I am also experiencing this same issue. 

TLDR: Followed https://landchad.net/nextcloud.html instructions.....twice.....no joy just get 404 error using "example.org/nextcloud" 

Interestingly enough, if I delete the "mywebsite" html link in the "/etc/nginx/sites-enabled" directory and try "example.org/nextcloud" I am greeted by Nextcloud setup/logon page....aka nextcloud seems to work if it is the only link in the said directory. 

As soon as I create a new "mywebsite" link following "https://landchad.net/nginx.html" it brings me back to the 404 page for "example.org/nextcloud" 

Makes me think that the html page "nextcloud" somehow needs adjusting to work with the(or any other html page) "mywebsite" page. 

I am hoping someone has an idea what to do with the "nextcloud" page or perhaps the author will respond to our errors/woes. 

Apologies if I have missed anything. I am a newb to Nextcloud and self-hosting.
